devito/dle/transformer.py

```diff
@@ -2,14 +2,14 @@
 
 from devito.archinfo import Platform, Cpu64
 from devito.ir.iet import Node
-from devito.dle.rewriters import CPU64Rewriter, CustomRewriter, State
+from devito.dle.rewriters import CPU64Rewriter, CustomRewriter, PlatformRewriter, State
 from devito.logger import dle as log, dle_warning as warning
 from devito.parameters import configuration
 
 __all__ = ['dle_registry', 'modes', 'transform']
 
 
-dle_registry = ('advanced', 'speculative')
+dle_registry = ('noop', 'advanced', 'speculative')
```

Comment:
Wait I thought there was a basic here at some points, examples may need update from this